### ISSUE TITLE
k8s not functional docker "apply caps: operation not permitted": unknown.

### ISSUE THREAD

**Issue by outbackdingo** [OverC/meta-overc#293]
so testing a few things out k8s wise... firstly... poky master meta-virtualization deploys k8s 1.18.3 can never seem to get networking up..... same with branch dunfell which deploys k8s 1.17.8 in each instance running 
(per the docs)  https://docs.windriver.com/bundle/Wind_River_Linux_OverC_Virtualization_Features_Guide_LTS_19/page/gno1576087591231.html

rm /opt/cni/bin
cp -a /usr/libexec/cni/ /opt/cni/bin
swapoff -a
kubeadm init --pod-network-cidr=10.244.0.0/16 --cri-socket /var/run/dockershim.sock

obviously dunfell and master use a later version already installed, the primary k8s comes up, however when i apply flannel
i start seeing failures

flannel Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.

and we have a loop.... making it unuseable

so we moved on to try something simple like rancher
sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 rancher/rancher

all went seemingly ok until we try to add worker nodes
sudo docker run -d --privileged --restart=unless-stopped --net=host -v /etc/kubernetes:/etc/kubernetes -v /var/run:/var/run rancher/rancher-agent:v2.4.5 --server https://192.168.1.60 --token 8742z4r8cgzc84p9r6c4f55s2b4mbmjcqtqlmbjb48hwh4hmfn7zx9 --ca-checksum 970ef02c3d0c65441f9a7c7522d606d15c592b09b498903e39ec57dac4bd6d19 --etcd --controlplane --worker

docker: Error response from daemon: OCI runtime create failed: container_linux.go:345: starting container process caused "apply caps: operation not permitted": unknown.

so simply put, has anyone of late deployed k8s on a more recent overc ... and if so how, or is there a secret to getting flannel to comply, or avoiding the docker apply_caps issue temporarily for testing.

**Comment by zeddii** [comment]
Sorry that I missed this when it came in. I'm cycling through meta-virt and updating the base components right now, I'll be back into containerized k8s once that is done, and will have more specific things to add.
